src/sentry/db/models/outboxes.py

```diff
@@ -365,6 +365,8 @@ def outbox_region_names(self) -> Collection[str]:
         """
         if hasattr(self, "organization_id"):
             return find_regions_for_orgs([self.organization_id])
+        if hasattr(self, "user_id"):
```

Comment:
Makes it a bit easier to infer the regions for a replicated model that have a user id associated.